### ISSUE TITLE
Capital letters in Microsoft.Maps.All.d.ts

### ISSUE THREAD

**Issue by RetsilaJ** [Microsoft/Bing-Maps-V8-TypeScript-Definitions#25]
Filename: Microsoft.Maps.All.d.ts

```
/// <reference path="Modules/GeoJson.d.ts"/>   on line 35 is without capital letters
```

file name is :  "GeoJSON.d.ts"

Fix:
```
/// <reference path="Modules/GeoJSON.d.ts"/>
```

**Comment by RetsilaJ** [comment]
I see its getting fixed in the september update:
https://github.com/Microsoft/Bing-Maps-V8-TypeScript-Definitions/milestone/2?closed=1

Issue can be closed

**Comment by RetsilaJ** [comment]
Wait a minute, September has passed!

**Comment by rbrundritt** [comment]
This has been resolved in version 2.0.1
